arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-deploying-token-bridge.md

```diff
@@ -0,0 +1,180 @@
+---
+title: 'How to Deploy a Token Bridge Using the Orbit SDK'
+sidebar_label: 'Token bridge deployment'
+description: 'How to deploy a token bridge using the Orbit SDK '
+author: Mehdi Salehi
+sme: Mehdi Salehi
+target_audience: 'Developers deploying and maintaining Orbit chains.'
+sidebar_position: 5
+---
+
+The <a data-quicklook-from='arbitrum-nitro'>Arbitrum Nitro stack</a> was designed without native support for specific token bridging standards at the protocol level, but Offchain Labs crafted a "canonical bridge" that ensures seamless token transfers between the parent and child chain. 
+
+This development enhances the interoperability of the Arbitrum Orbit ecosystem. 
+The token bridge architecture includes contracts located on the  <a data-quicklook-from='parent-chain'>parent chain</a> as well as a complementary set of contracts on the <a data-quicklook-from='child-chain'>child chain</a>. These entities communicate via the <a data-quicklook-from='retryable-ticket'>Retryable Ticket </a> protocol, ensuring efficient and secure interactions. 
+
+For an in-depth exploration of the token bridge mechanism, feel free to consult the [`ERC-20` token bridge overview](https://docs.arbitrum.io/for-devs/concepts/token-bridge/token-bridge-erc20), which provides insights into the bridge's design and operational dynamics.
+
+### Token Bridge Deployment Steps
+
+Following the [deployment](orbit-sdk-deploying-token-bridge.md) and initialization of the Orbit chain, the subsequent phase involves deploying contracts on both the parent and child chains. 
```

Comment:
I think the link should point to any of the create-rollup examples (it's pointing at this page instead)

Reply:
Merged manually.